src/components/DataTable/DataTable.jsx

```diff
@@ -178,6 +178,13 @@ const DataTable = createClass({
 		};
 	},
 
+	getInitialState() {
```

Comment:
Wonder if this qualifies for an hybrid component?

https://appnexus.github.io/lucid/?selectedKind=Lucid%20UI&selectedStory=Hybrid%20State%20Components&full=0&addons=0&stories=1&panelRight=0&addonPanel=lucid-docs-panel-props